src/main/resources/hudson/tasks/test/AbstractTestResultAction/widget.jelly

```diff
@@ -0,0 +1,66 @@
+<!--
+The MIT License
+
+Copyright (c) 2025, Jan Faracik
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
+    <l:card title="${it.displayName}" expandable="${it.urlName}">
+        <j:set var="passCount" value="${it.passedTests.size()}" />
+        <j:set var="skipCount" value="${it.skipCount}" />
+        <j:set var="failCount" value="${it.failCount}" />
+        <j:set var="totalCount" value="${it.totalCount}" />
+        <j:set var="duration" value="${it.result.durationString}" />
+
+        <j:if test="${it.failCount gt 0}">
+            <a class="jenkins-card__teaser" href="${it.urlName}" tabindex="-1">
+                <l:icon src="symbol-close-circle-outline plugin-ionicons-api"
+                        class="jenkins-!-error-color" />
+                <div>${failCount} failing ${failCount eq 1 ? 'test' : 'tests'}</div>
+
+                <div>
+                    ${passCount} passing, ${skipCount} skipped, ${totalCount} ran
```

Comment:
This string should be i18n with parameters for the place holders, applies throughout